Tools/autotest/pysim/util.py

```diff
@@ -815,6 +815,13 @@ def load_local_module(fname):
     return ret
 
 
+def get_git_hash():
+    githash = run_cmd('git rev-parse HEAD', output=True, directory=reltopdir('.')).strip()
```

Comment:
```suggestion
    githash = run_cmd('git rev-parse --short=8 HEAD', output=True, directory=reltopdir('.')).strip()
```

Maybe use `--short=8` so the repo hash is the same length as the firmware hash?

Reply:
yep, will make comparison easier, thx